app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -124,12 +124,24 @@ def analyze
                           msg % [options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]],
                           'error')
     end
+
+    collect_compliance_data(image)
+
     image.scan_metadata(SCAN_CATEGORIES,
                         "taskid" => jobid,
                         "host"   => MiqServer.my_server,
                         "args"   => [YAML.dump(scan_args)])
   end
 
+  def collect_compliance_data(image)
+    return unless image_inspector_client.respond_to?(:fetch_oscap_arf)
+
+    open_scap_result = OpenscapResult.new(:container_image => image)
+    open_scap_result.raw = image_inspector_client.fetch_oscap_arf
+
+    open_scap_result.save
```

Comment:
@simon3z Having separate scan and sync phases would be more consistent with our existing SSA state machine, but given that the OpenSCAP scanning differs from standard SSA in a number of ways, I'm not sure if it matters enough to change it at this point. I say leave it the way it is, and we'll address any unforeseen issues if we encounter them.